server/channels/app/authorization.go

```diff
@@ -206,7 +196,7 @@ func (a *App) SessionHasPermissionToChannelByPost(session model.Session, postID
 		return false
 	}
 
-	if channelMember, err := a.Srv().Store().Channel().GetMemberForPost(postID, session.UserId, *a.Config().TeamSettings.ExperimentalViewArchivedChannels); err == nil {
+	if channelMember, err := a.Srv().Store().Channel().GetMemberForPost(postID, session.UserId, true); err == nil {
```

Comment:
We're not currently removing the ability to filter archived channels, just hard-coding the value now that the config isn't meant to be used. (We may selectively restore the ability to filter out archived channels in part of the UI going forward.)

Reply:
Can we note this in a comment for posterity?